src/kernel/include/pgtbl.h

```diff
@@ -279,6 +294,31 @@ pgtbl_mapping_add(pgtbl_t pt, u32_t addr, u32_t page, u32_t flags)
 	return ret;
 }
 
+/* TODO: remove. a hack for kmem. */
```

Comment:
Explain why it is a hack, and what the proper solution would be.  It isn't obvious to me.